modules/landing-pages/module.php

```diff
@@ -212,53 +269,169 @@ private function register_landing_page_cpt() {
 			'show_in_menu' => 'edit.php?post_type=elementor_library&tabs_group=library',
 			'capability_type' => 'page',
 			'taxonomies' => [ Source_Local::TAXONOMY_TYPE_SLUG ],
+			'hierarchical' => true,
```

Comment:
Why?

Reply:
I thought we discussed this could help when checking for identical permalinks.

Reply:
It not help.

Reply:
Removed.